google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionClient.java

```diff
@@ -237,6 +237,19 @@ SessionImpl createSession() {
    * @param consumer The {@link SessionConsumer} to use for callbacks when sessions are available.
    */
   void createMultiplexedSession(SessionConsumer consumer) {
+    try {
+      SessionImpl sessionImpl = createMultiplexedSession();
+      consumer.onSessionReady(sessionImpl);
+    } catch (Throwable t) {
+      consumer.onSessionCreateFailure(t, 1);
+    }
+  }
+
+  /**
+   * Create a multiplexed session and returns it. A multiplexed session is not affiliated with any
+   * GRPC channel. In case of an error on the gRPC calls, the error will be returned.
```

Comment:
```suggestion
   * Creates a multiplexed session and returns it. A multiplexed session is not affiliated with any
   * GRPC channel. In case of an error during the gRPC calls, an exception will be thrown.
```